lib/ansible/module_utils/network/iosxr/argspec/acl_interfaces/acl_interfaces.py

```diff
@@ -0,0 +1,85 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+"""
+The arg spec for the iosxr_acl_interfaces module
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class Acl_interfacesArgs(object):  # pylint: disable=R0903
+    """The arg spec for the iosxr_acl_interfaces module
+    """
+    def __init__(self, **kwargs):
+        pass
+
+    argument_spec = {
+        'running_config': {
+            'type': 'str'
+        },
+        'config': {
+            'elements': 'dict',
+            'options': {
+                'access_groups': {
+                    'elements': 'dict',
+                    'options': {
+                        'acls': {
+                            'elements': 'dict',
+                            'options': {
+                                'direction': {
+                                    'choices': ['in', 'out'],
+                                    'type': 'str'
+                                },
+                                'name': {
+                                    'type': 'str'
+                                }
+                            },
+                            'type': 'list'
+                        },
+                        'afi': {
```

Comment:
IMO it should be mandatory

Reply:
We cannot make `afi` mandatory because we can have a `delete` task as follows:
```
- iosxr_acl_interfaces:
     config:
         - name: GigabitEthernet0/0/0/1
     state: deleted
```

Reply:
I think we can because afi is an attribute of access_group so that doesn't affect the above scenario.

Reply:
Ah, yes. You're correct.